test/datatypes.test.js

```diff
@@ -37,15 +35,11 @@ describe('MySQL specific datatypes', function () {
     });
   });
 
-  it('should fail spectacularly with invalid enum values', function (done) {
-    var em = EnumModel.create({animal: 'horse', condition: 'sleepy', mood: 'happy'}, function (err, obj) {
-      assert.ok(!err);
-      EnumModel.findById(obj.id, function (err, found) {
-        assert.ok(!err);
-        assert.equal(found.animal, ''); // MySQL fun.
-        assert.equal(found.animal, 0);
-        done();
-      });
+  it('should fail with invalid enum values', function (done) {
+    EnumModel.create({animal: 'horse'}, function(err) {
+      assert.ok(err);
+      assert.equal(err.errno, 1265);
```

Comment:
Yes this is MySQL-version specfiic, but I'm going to close this PR as juggler tests already pass on master.